client/web/compose/src/components/PageBlocks/ChartBase.vue

```diff
@@ -4,6 +4,60 @@
     v-on="$listeners"
     @refreshBlock="refresh"
   >
+    <template>
+      <b-button
+        rounded="full"
+        variant="outline-light"
+        class="chart__livefilter-btn text-secondary d-print-none"
+        @click="liveFilterModal.show = true"
+      >
+        <font-awesome-icon
+          :icon="['fas', 'filter']"
+        />
+      </b-button>
+
+      <b-modal
+        v-model="liveFilterModal.show"
+        :title="liveFilterModal.title"
+        :ok-title="$t('general:label.saveAndClose')"
+        centered
+        size="md"
+        cancel-variant="light"
+        no-fade
+        @ok="liveFilterUpdate()"
+        @cancel="liveFilterModal.show = undefined"
+        @hide="liveFilterModal.show = undefined"
+      >
+        <b-form-group
+          :label="$t('chart:edit.filter.preset')"
+          label-class="text-primary"
+        >
+          <c-input-select
+            v-model="liveFilterValue"
+            :options="predefinedFilters"
+            label="text"
+            :reduce="filter => filter.value"
+            :placeholder="$t('chart:edit.filter.noFilter')"
+          />
+        </b-form-group>
+
+        <template #modal-footer="{ ok }">
+          <b-button
+            variant="light"
+            @click="resetLiveFilter()"
+          >
+            Reset
```

Comment:
We need a cancel button aswell
Also move reset to the left of the modal (similar to record list filter)